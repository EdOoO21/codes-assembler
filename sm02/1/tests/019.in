-912 607 833 65 -753 47 -592 -195 -289 83 -408 197 -697 218 -466 -780 -467 -157 -330 982 -452 709 -780 583 -334 837 -363 -954 165 264 -588 800 -856 884 -586 -767 110 -67 -205 773 502 -850 644 -799 -152 -966 -797 185 490 -134 547 746 -189 -99 348 -388 168 873 28 707 -676 621 304 953 -168 -524 586 116 180 497 -780 641 -576 -201 -608 -717 -725 282 10 -786 120 444 -62 -986 959 -70
