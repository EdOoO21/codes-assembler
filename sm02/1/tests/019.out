-70
959
-986
-62
444
120
-786
10
282
-725
-717
-608
-201
-576
641
-780
497
180
116
586
-524
-168
953
304
621
-676
707
28
873
168
-388
348
-99
-189
746
547
-134
490
185
-797
-966
-152
-799
644
-850
502
773
-205
-67
110
-767
-586
884
-856
800
-588
264
165
-954
-363
837
-334
583
-780
709
-452
982
-330
-157
-467
-780
-466
218
-697
197
-408
83
-289
-195
-592
47
-753
65
833
607
-912
